src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -136,6 +136,11 @@ def step(
         predict_epsilon=True,
         generator=None,
     ):
+        if self.num_inference_steps is None:
```

Comment:
Here I don't think we need the error message as DDPM usually runs with number of training steps